client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -234,6 +234,7 @@ const Organization = ({ organizationId }: { organizationId: string }) => {
             <input
               type="text"
               name="name"
+              id="name"
```

Comment:
testing library upgrade caught that this input needed an `id` in order to be associated correctly with its label